easybuild/easyconfigs/s/Salmon/Salmon-0.8.2-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '0.8.2'
+
+homepage = 'https://github.com/COMBINE-lab/salmon'
+description = """Salmon is a wicked-fast program to produce a highly-accurate,
+ transcript-level quantification estimates from RNA-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/COMBINE-lab/salmon/archive/']
+
+checksums = ['299168e873e71e9b07d63a84ae0b0c41b0876d1ad1d434b326a5be2dce7c4b91']
+
+patches = ['Salmon-%(version)s_CMAKE_CXX_FLAGS.patch']
+
+python = 'Python'
+pyver = '2.7.12'
+versionsuffix = '-%s-%s' % (python, pyver)
```

Comment:
the `python` and `pyver` variables aren't used anywhere, so please drop them, and move this up right below the `version =` line:

```python
versionsuffix = '-Python-2.7.12'
```